x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -2756,7 +2781,7 @@ describe('Task Runner', () => {
 
     expect(logger.debug).nthCalledWith(
       3,
-      'Rule "1" skipped scheduling action "1" because the maximum number of allowed actions for connector type .server-log has been reached.'
+      'Rule "1" skipped scheduling action "2" because the maximum number of allowed actions for connector type .server-log has been reached.'
```

Comment:
I changed the value in this test, but I want to make sure that I didn't mess with the circuit breaker by adding the chunking and bulk enqueue. This seems to make sense to me if the limit is one action for the `.server_log` connector type then it would skip scheduling action 2. If someone wouldn't mind double checking this, that'd be great 🙂

Reply:
Verified this is working as expected